tests/x509/test_x509.py

```diff
@@ -795,6 +795,10 @@ def test_non_ascii_dns_name(self, backend):
             u'xn--biztosts-fza2j.hu', u'*.xn--biztosts-fza2j.hu'
         ]
 
+    def test_non_ascii_rfc822name(self, backend):
+        # TODO
+        pass
```

Comment:
...

Reply:
Oh yeah 😄  oops, that test turns out to not be needed but I forgot to remove it.